### PR TITLE
JavaScript-II

### PR DESCRIPTION
**Pull request by KarimBertacche**
Complete main challenges for arrays methods, closure, and callback function passed to higher order functions

### CODE REVIEW

**Comment by AnthonyJCampbell** [comment]
Karim,
All in all, a great job! Despite a couple of minor hiccups here and there, you pretty much blew through the assignment at light speed! Congrats, dude!

Technically, your code is great. Good use of both fat-arrow functions as well as chaining the array methods. It took me quite a bit longer than you to figure I could do that. What's more, you actually _understand_ what every intermediate step is doing. Even more than being able to just type the code, that's what makes one a great developer!

I _am_ going to keep harassing you about committing and pushing to Github more often, but just a little bit. 😄😉  However, if that's the only thing I can provide as feedback, that should signal that you did a pretty damn good job! 

In summary: great work, keep it up, and push to Github! 😂 🚀
